src/lib/constants.ts

```diff
@@ -482,3 +493,19 @@ export const lastTripCache = new Map<
 	string,
 	{ continue: (message: KlasaMessage) => Promise<KlasaMessage | KlasaMessage[] | null>; data: ActivityTaskOptions }
 >();
+
+// Override Roles and Channels settings with what is set on the config.ts
+if (DISCORD_SETTINGS.Roles) {
+	for (const [_key, _value] of objectEntries(DISCORD_SETTINGS.Roles)) {
+		if ((Roles as any)[_key]) {
+			(Roles as any)[_key] = _value;
+		}
+	}
+}
+if (DISCORD_SETTINGS.Channels) {
+	for (const [_key, _value] of objectEntries(DISCORD_SETTINGS.Channels)) {
+		if ((Channel as any)[_key]) {
+			(Channel as any)[_key] = _value;
+		}
+	}
+}
```

Comment:
Cant you just do this in the declaration? `SupportChannel = Roles.SupportChannel ?? '123123123';`

Reply:
No, you cant do that with enums.

Reply:
we need to do something a bit better, this looks unholy - just change it to be a plain object if needed.

Reply:
Done, changed it from enum to an object and changing it on object init.